internal/cli/execute.go

```diff
@@ -146,13 +146,15 @@ func runExecute(cmd *cobra.Command, opts executeOptions) error {
 
 	sauce, err := re.Execute(engine.Engine{}, values, uuid.Must(uuid.NewV4()))
 	if err != nil {
-		return err
+		retryMessage := makeRetryMessage(values)
+		return fmt.Errorf("%w\n\n%s", err, retryMessage)
 	}
 
 	// Check for conflicts
 	for _, s := range existingSauces {
 		if conflicts := s.Conflicts(sauce); conflicts != nil {
-			return fmt.Errorf("conflict in recipe '%s': file '%s' was already created by recipe '%s'", re.Name, conflicts[0].Path, s.Recipe.Name)
+			retryMessage := makeRetryMessage(values)
```

Comment:
Could we also add this to `upgrade` command?

Reply:
Sure! Should be no problem now that the message generator function doesn't depend on anything.

Reply:
It's added to a bunch of places in upgrade now too.